docs/reference/data-streams/lifecycle/apis/get-global-retention.asciidoc

```diff
@@ -0,0 +1,89 @@
+[[data-streams-get-global-retention]]
+=== Get the global retention of data streams
+++++
+<titleabbrev>Get Data Stream Global Retention</titleabbrev>
+++++
+
+preview::[]
+
+Gets the global retention settings that apply on every data streams managed by data stream lifecycle.
+
+[[get-global-retention-api-prereqs]]
+==== {api-prereq-title}
+
+** If the {es} {security-features} are enabled, you must have the `monitor_data_stream_global_retention` or
+`manage_data_stream_global_retention` <<privileges-list-cluster,cluster privilege>> to use this API.
+
+[[data-streams-get-global-retention-request]]
+==== {api-request-title}
+
+`GET _data_stream/_global_retention`
+
+[[data-streams-get-global-retention-desc]]
+==== {api-description-title}
+
+Gets the global retention settings that applied on data streams managed by data stream lifecycle.
+
+[role="child_attributes"]
+[[get-global-retention-api-query-parms]]
+==== {api-query-parms-title}
+
+`local`::
+(Boolean) Specifies whether to retrieve local information. The default value
+is `false`, which means the information is retrieved from the master node.
+
+[[get-global-retention-api-response-body]]
+==== {api-response-body-title}
+
+`default_retention`::
+(Optional, string)
+The default retention that will apply to any data stream managed by data stream lifecycle that does not have a retention
+defined on the data stream level.
+
+`max_retention`::
+(Optional, string)
+The max retention that will apply to all data stream managed by data stream lifecycle. The max retention will override the
+retention of a data stream which retention exceeds the max retention.
```

Comment:
I think "The max retention will override the retention of a data stream *whose* retention exceeds the max retention." would be more grammatically correct.